hls4ml/backends/vivado/passes/match_quantizer_resize.py

```diff
@@ -0,0 +1,16 @@
+from hls4ml.model.layers import Resize
+from hls4ml.model.optimizer import OptimizerPass
+
+def register_match_quantizer_resize(backend):
```

Comment:
I don't think we need a second optimizer that does the work meant to be done by infer-precision

Reply:
I'd generally agree but this is for the specific implementation of resize nearest for vivado/vitis. Other type of resize algorithm may require different type of type inference